run_dir/static/js/projects.js

```diff
@@ -631,10 +631,10 @@ function updateStatusBar1(source, type){
       dealWithDatepickers('datepick3', 'add');
       dealWithDatepickers('datepick2', 'add');
     }
-    if(jQuery.inArray('statusOptOngoing', checklist) !== -1 || jQuery.inArray('statusOptOpen', checklist) !== -1 || jQuery.inArray('statusOptNeedsReview', checklist) !== -1 || jQuery.inArray('statusOptRecCtrl', checklist) !== -1 ){
```

Comment:
This seems like a kind of random change - is it intentional?

Reply:
It was a minor bug which I noticed while testing.